app/core/store/modules/prepaids.js

```diff
@@ -114,11 +114,15 @@ export default {
   },
 
   actions: {
-    fetchPrepaidsForTeacher: ({ commit }, { teacherId, sharedClassroomId }) => {
+    fetchPrepaidsForTeacher: ({ commit }, { teacherId, sharedClassroomId, includeShared }) => {
       commit('toggleLoadingForTeacher', teacherId)
 
+      if (includeShared === undefined) {
```

Comment:
can't we do `{ teacherId, sharedClassroomId, includeShared = true }` in params?

Reply:
emmm maybe? it is js so it should work 😂